Targets/Extensions/watchOS-Extension.xcconfig

```diff
@@ -0,0 +1,19 @@
+//
+// watchOS-Extension.xcconfig
+//
+// Copyright (c) 2016 Twitter. All rights reserved.
+//
+
+#include "ExtensionTarget.xcconfig"
+
+SUPPORTED_PLATFORMS = watchsimulator watchos
+
+SDKROOT = watchos
+
+CODE_SIGN_IDENTITY = iPhone Developer
```

Comment:
🔥

On Tuesday, June 7, 2016, Andrew McKnight <notifications@github.com> wrote:

> In Targets/Extensions/watchOS-Extension.xcconfig
> <https://github.com/twitter-fabric/FABConfig/pull/12#discussion_r66168019>
> :
>
> > @@ -0,0 +1,19 @@
> > +//
> > +// watchOS-Extension.xcconfig
> > +//
> > +// Copyright (c) 2016 Twitter. All rights reserved.
> > +//
> > +
> > +#include "ExtensionTarget.xcconfig"
> > +
> > +SUPPORTED_PLATFORMS = watchsimulator watchos
> > +
> > +SDKROOT = watchos
> > +
> > +CODE_SIGN_IDENTITY = iPhone Developer
>
> Actually, Target.xcconfig already includes it, so I will remove this
> setting from here altogether
>
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub
> <https://github.com/twitter-fabric/FABConfig/pull/12/files/8e68e57a8ca1b8e6226a9eea208063e976d777c0#r66168019>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AAbRqbNu-7yTB6JKQ_Q4QOYJkMDuPyCNks5qJfiWgaJpZM4Iv_4Y>
> .
>